src/backend/src/prisma/migrations/20240129020615_rr_identifier/migration.sql

```diff
@@ -0,0 +1,26 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[identifier]` on the table `Reimbursement_Request` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- AlterTable
+ALTER TABLE "Reimbursement_Request" ADD COLUMN     "identifier" SERIAL NOT NULL;
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Reimbursement_Request_identifier_key" ON "Reimbursement_Request"("identifier");
+
+-- Assign Identifier to all old Reimbursement Requests based on their dateCreated order
```

Comment:
I got this SQL code from ChatGPT (chat: https://chat.openai.com/share/d0692ab8-4656-44bd-b06e-3c99511ac1fc) after providing the context of our RR model and situation.